tutorials/07-implementing-a-physics-plugin.md

```diff
@@ -7,8 +7,8 @@ This tutorial shows how to develop a simple plugin that implements a
 ## Prerequisites
 
 - \ref installation "Installation"
-- \ref physicsplugin "Understand physics plugin"
-- \ref loadplugin "Load physics plugin"
+- \ref physicsplugin "Understanding the physics plugin"
+- \ref pluginloading "Loading physics plugins"
```

Comment:
Changing the titles of these tutorials is fine, but I don't think we want to change the refs because the urls use those and changing them now would break existing url references and search engine results.

Reply:
These aren't changes. They are actually fixes. The refs right now are completely broken. It looks like Claire started cleaning up the tutorials but didn't get a chance to finish, so I pattern-matched the titles to the ones she started cleaning up. The refs look like they were just never finalized - they changed the labels in the actual file, but didn't update places that reference to them.

Reply:
Ok, got it. I also just did a google search, and the url for "Loading physics plugins" corresponds to `pluginloading`, not `loadplugin`.